projects/igniteui-angular/src/lib/list/list.component.ts

```diff
@@ -125,18 +230,31 @@ export class IgxListComponent {
 
         return headers;
     }
-
+    /**
+     * Returns a reference to `IgxListComponent` element in the DOM.
```

Comment:
Returns the `context` object which represents the `template context` binding into the `list container` by providing the `$implicit` declaration which is the `IgxListComponent` itself.